crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -218,6 +273,48 @@ impl IndexerStore for PgIndexerStore {
         })
     }
 
+    fn get_transaction_digest_page_by_move_call(
+        &self,
+        package_name: String,
+        module_name: Option<String>,
+        function_name: Option<String>,
+        start_sequence: i64,
+        limit: usize,
+        reverse: bool,
+    ) -> Result<Vec<String>, IndexerError> {
+        let mut pg_pool_conn = get_pg_pool_connection(&self.cp)?;
+        pg_pool_conn
+            .build_transaction()
+            .read_only()
+            .run(|conn| {
+                let mut builder = move_calls_dsl::move_calls.filter(move_calls_dsl::move_package.eq(package_name.clone())).into_boxed();
+                if let Some(module_name) = module_name.clone() {
+                    builder = builder.filter(move_calls_dsl::move_module.eq(module_name));
+                }
+                if let Some(function_name) = function_name.clone() {
+                    builder = builder.filter(move_calls_dsl::move_function.eq(function_name));
+                }
+                if reverse {
```

Comment:
Aside from the potential duplication problem above, which is within a single request, can there be an edge case where there could be duplicate data across adjacent requests with the second requests uses the cursor from previous request?

Reply:
that's very possible but I did not see issues with that, as the cursor from previous request was not in the previous page, can you elaborate?